src/v2/Apps/ArtAppraisals/ArtAppraisalsApp.tsx

```diff
@@ -91,7 +91,7 @@ const Header: React.FC = () => {
 
 const WhyUseArtsy: React.FC = () => {
   const { src, srcSet } = resized(
-    "https://files.artsy.net/images/appraisals-main-image.png",
+    "https://files.artsy.net/images/shinpei-kusanagi-stop-talking-to-yourself-you-will-be-alright-number-3.jpeg",
     {
       width: 2070,
       height: 2160,
```

Comment:
Where'd you get these dimensions from? Should be 850x887 from the looks of it.

Reply:
I just multiplied by 2 or 3 because you were saying that ideally the images should be real big. Lol, clearly i still do not understand how to handle images on the web 😦

Reply:
will update.

Reply:
:)

`srcSet` handles the 2x-ing. You want to just inspect it at the max breakpoint and use those dimensions.

Reply:
The thing we want to be really big is the source file which we use to pull resizes/crops from.